src/dos/dos_keyboard_layout.cpp

```diff
@@ -170,7 +161,7 @@ void KeyboardLayout::Reset()
 	diacritics_entries=0;		// no diacritics loaded
 	diacritics_character=0;
 	user_keys=0;				// all userkeys off
-	language_code_count=0;
+	language_codes = {};
```

Comment:
`std::list` has a `clear()` method for this

Reply:
Thanks; added.